runtime/jvmti/jvmtiHook.c

```diff
@@ -505,6 +509,56 @@ jvmtiHookVirtualThreadUnmount(J9HookInterface **hook, UDATA eventNum, void *even
 
 #endif /* JAVA_SPEC_VERSION >= 19 */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static void
+jvmtiHookVMCheckpoint(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData)
+{
+	J9CheckpointEvent * data = eventData;
+	J9JVMTIEnv * j9env = userData;
+	jvmtiExtensionEvent callback = *J9JVMTI_EXTENSION_CALLBACK(j9env, J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT);
+
+	Trc_JVMTI_jvmtiHookVMCheckpoint_Entry();
+
+	if (NULL != callback) {
+		J9VMThread * currentThread = data->currentThread;
+		jthread threadRef = NULL;
+		UDATA hadVMAccess = 0;
+		UDATA javaOffloadOldState = 0;
+
+		if (prepareForEvent(j9env, currentThread, currentThread, J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT, &threadRef, &hadVMAccess, FALSE, 0, &javaOffloadOldState)) {
+			callback((jvmtiEnv *) j9env, (JNIEnv *) currentThread, threadRef);
+			finishedEvent(currentThread, J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT, hadVMAccess, javaOffloadOldState);
+		}
+	}
+
+	TRACE_JVMTI_EVENT_RETURN(jvmtiHookVMCheckpoint);
+}
+
+static void
+jvmtiHookVMRestore(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData)
```

Comment:
The same changes for the pointer symbol(*) as requested at https://github.com/eclipse-openj9/openj9/pull/17252#discussion_r1175832000.